server/src/test/java/org/opensearch/indices/store/IndicesStoreTests.java

```diff
@@ -100,6 +100,34 @@ public void testShardCanBeDeletedNoShardStarted() throws Exception {
         assertFalse(IndicesStore.shardCanBeDeleted(localNode.getId(), routingTable.build()));
     }
 
+    public void testShardCanBeDeletedWithRelocating() throws Exception {
```

Comment:
Do you need to update NOT_STARTED_STATES in the previous test to NOT_ACTIVE_STATES? Else, it can fail intermittently?

Reply:
Thanks for catching it. I have corrected it with active states.